packages/core/src/layout/InfoCard/BottomLink.tsx

```diff
@@ -0,0 +1,65 @@
+/*
+ * Copyright 2020 Spotify AB
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import React, { FC } from 'react';
+import { Link } from '@material-ui/core';
+import { Divider, ListItemText, makeStyles } from '@material-ui/core';
+import { ListItem, ListItemIcon } from '@material-ui/core';
+import ArrowIcon from '@material-ui/icons/ArrowForward';
+import grey from '@material-ui/core/colors/grey';
+import Box from '@material-ui/core/Box';
+import { BackstageTheme } from '../../theme/theme';
+
+const useStyles = makeStyles<BackstageTheme>(theme => ({
+  root: {
+    maxWidth: 'fit-content',
```

Comment:
I hadn't seen this one before. Support is not entirely there, mozilla needs a prefix right? And does not this one need a parameter percentage too?

https://developer.mozilla.org/en-US/docs/Web/CSS/max-width